programs/fileio.c

```diff
@@ -683,7 +683,7 @@ FIO_openDstFile(FIO_ctx_t* fCtx, FIO_prefs_t* const prefs,
         }
 #endif
         if (!prefs->overwrite) {
-            if (g_display_prefs.displayLevel <= 1) {
+            if (g_display_prefs.displayLevel == 0) {
```

Comment:
As the comment implies, the goal here is to take over decision when prompt interactions are disabled.
`-q` disables user interactions.
It's necessary for some script to not "hang" waiting for user input.

So the previous threshold `<= 1` was valid.

The condition could be rewritten differently, in order to be more understandable.
But it's nonetheless correct.

Reply:
We  have some CI tests, and I could imagine some scripts use this behavior, where stderr is not a tty, however we pipe in "y" or "yes" to continue execution.  With this change, those are now considered `-q` and fail if we disable user interactions.  I suppose in those use cases, we could just require that `-f` be used instead and we could remove those test cases.  Or, we could change this change so that rather than fully assuming `-q` when stderr is tty, we instead leave the display level alone and more specifically target the progress bar and suppress it.